examples/ai-ml/ray/sources/serve_fashion_mnist.py

```diff
@@ -0,0 +1,81 @@
+import ray
+from ray import serve
+import torch
+from ray.train.torch import TorchPredictor, TorchCheckpoint
+from transformers import AutoConfig, AutoModelForSequenceClassification, AutoTokenizer
+import tempfile
+import s3fs
+import boto3
+
+
+@serve.deployment(route_prefix="/predict", version="0.1.0")
+class Predictor:
+
+    def __init__(self):
+        self.classes = [
+            "Negative",
+            "Neutral",
+            "Positive",
+        ]
+
+        s3_file = s3fs.S3FileSystem()
+        ssm = boto3.client("ssm")
+        s3_path = ssm.get_parameter(Name="/ray-demo/model_checkpoint")["Parameter"]["Value"]
```

Comment:
interesting that they checkpoint to SSM and not S3

Reply:
That's our poor man's model registry for the moment :)